model/flow/identifierList.go

```diff
@@ -12,7 +12,7 @@ import (
 type IdentifierList []Identifier
 
 // Len returns length of the IdentiferList in the number of stored identifiers.
-// It satisfies the sort.Interface making the IdentifierList sortable.
+// It satisfies the sort.Type making the IdentifierList sortable.
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig